bin/v-add-web-domain-redirect

```diff
@@ -42,18 +42,22 @@ is_object_unsuspended 'user' 'USER' "$user"
 is_object_valid 'web' 'DOMAIN' "$domain"
 is_object_unsuspended 'web' 'DOMAIN' "$domain"
 
-idn_redirect=$(idn2 --quiet $3)
-if [ $? == 0 ]; then
-	redirect=$idn_redirect
-fi
-
 if [[ "$3" =~ http://|https:// ]]; then
-	scheme=1
+	scheme_check=1
+	scheme=$($HESTIA_PHP -r '$url=parse_url($argv[1]); echo $url["scheme"];' "$redirect")
+	host=$($HESTIA_PHP -r '$url=parse_url($argv[1]); echo $url["host"];' "$redirect")
+	path=$($HESTIA_PHP -r '$url=parse_url($argv[1]); if(!empty($url["path"])){echo $url["path"];}' "$redirect")
+	host=$(idn2 --quiet $host)
```

Comment:
better do
```
host=$(idn2 --quiet "$host")
```
just to be on the safe side

Reply:
Changed :)